railties/lib/rails/generators/rails/app/templates/config/puma.rb

```diff
@@ -0,0 +1,9 @@
+threads_count = ENV.fetch("MAX_THREADS") { 5 }.to_i
```

Comment:
I'm not sure about this ENV-all-the-things.. that's really not how we configure Rails.